text/0161-telemetry-schema-and-resource.md

```diff
@@ -0,0 +1,128 @@
+# Telemetry Schema and Resource
+
+Specify a mechnism for users of Resource to interact with Resource leveraging
+TelemetrySchema version for a stable API.
+
+## Motivation
```

Comment:
Could you add a couple of examples (real or made-up) illustrating the kinds of breaks that this OTEP would protect/mitigate?

Reply:
done